### PR TITLE
add often/rua to the db

### PR DESCRIPTION
**Pull request by often**
"rua" is a random user-agent generator module for Deno, it aims to be simple, light and use-effective.

### CODE REVIEW

File: database.json [hunk 1]
```diff
@@ -1186,6 +1192,12 @@
     "repo": "dso",
     "desc": "dso is a simple ORM Library based on deno_mysql"
   },
+  "duckduckgo": {
+    "type": "github",
+    "owner": "0vw",
+    "repo": "duckduckgo",
+    "desc": "A Deno wrapper for the DuckDuckGo API"
```
Comment (ry) [inline database.json:1199]: Unrelated changes. Please merge with master

Comment (often) [reply database.json]: I have updated it according to the latest database.json
I am not sure what you mean by "Please merge with master", I assume that?
Let me know further

File: database.json [hunk 2]
```diff
@@ -53,6 +53,12 @@
     "repo": "api",
     "desc": "Promise based HTTP client for the Deno. ༼ つ ◕_◕ ༽つ"
   },
+  "appwrite": {
+    "type": "github",
+    "owner": "appwrite",
+    "repo": "sdk-for-deno",
+    "desc": "Official Deno SDK for the Appwrite open-source backend server 🚀"
+  },
```
Comment (ry) [inline database.json:61]: not ok. I'll close the PR, just try with a fresh branch.

Comment (often) [reply database.json:61]: Sorry, I'm still getting a hang with pull requests
Sure, will do :)
